docs/release-info/release-notes.md

```diff
@@ -462,6 +491,12 @@ You can now optionally use the `caseSensitive` Boolean config to configure how D
 
 [#16496](https://github.com/apache/druid/pull/16496)
 
+#### Iceberg REST catalog support
+
+Adds support to the `iceberg` input source to read from Iceberg REST catalogs.
+
+[#17124](https://github.com/apache/druid/pull/17124)
+
```

Comment:
Removing the iceberg entry and will manually add it to the extensions section since iceberg support ships as a community extension 
```suggestion
```